appium/tests/screenobjects/old-version-app.screen.ts

```diff
@@ -4,6 +4,7 @@ import ElementHelper from "../helpers/ElementHelper";
 const SELECTORS = {
   BACK_BUTTON: '~arrow left c',
   SENDER_EMAIL: '~messageSenderLabel',
+  PUBLIC_KEY: '-ios class chain:**/XCUIElementTypeOther/XCUIElementTypeStaticText[2]'
```

Comment:
should have aid for this - can file a separate issue

Reply:
In old app we dont have aid for this element

Reply:
We can't be restricted by that. The old app should use the old selector, but we should use the new selector for the new app.

Reply:
for new app we use new selector
check `PublicKeyDetailsScreen`
<img width="424" alt="Screenshot 2022-01-21 at 17 15 51" src="https://user-images.githubusercontent.com/91205853/150551797-25b96697-a722-456d-8dc4-153aeb28130e.png">